.circleci/config.yml

```diff
@@ -1298,7 +1298,7 @@ workflows:
       - fe-tests-cypress:
           matrix:
             parameters:
-              node: ["1", "2", "3", "4"]
+              node: ["1"]
               edition: ["ee", "oss"]
           name: fe-tests-cypress-<< matrix.node >>-<< matrix.edition >>
           requires:
```

Comment:
I would still include the change to change `cypress-group` to include `ee`/`oss` here e.g.

```yaml
cypress-group: "default-<< matrix.edition >>"
```

I don't know if Cypress.io was smart enough to know they were different groups but I'd rather be safe than sorry. This way we don't forget to do it whenever we reenable Cypress

Reply:
Yeah good call. It certainly doesn't hurt to include it!